src/allmydata/test/test_util.py

```diff
@@ -553,6 +553,17 @@ def test_any_bytes(self):
                 o, cls=jsonbytes.AnyBytesJSONEncoder)),
             expected,
         )
+        self.assertEqual(
+            json.loads(jsonbytes.dumps(o, any_bytes=True)),
+            expected
+        )
+
+    def test_dumps_bytes_unicode_separators(self):
+        """Unicode separators don't prevent the result from being bytes."""
+        result = jsonbytes.dumps_bytes([1, 2], separators=(u',', u':'))
```

Comment:
would some non-ascii unicode cases also be worth exercising here?

Reply:
this was probably actually about `unicode` vs `bytes` (i.e. py2 test)

Reply:
(is that right @itamarst ?)